api/resources_portal/management/commands/material_additional_metadata_key_change.py

```diff
@@ -0,0 +1,183 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+from resources_portal.config.logging import get_and_configure_logger
+from resources_portal.models import Material
+
+logger = get_and_configure_logger(__name__)
+
+
+def validate_category(category):
+    return len([choice for choice in Material.CATEGORY_CHOICES if category in choice]) > 0
+
+
+def paginate_queryset(query_set, function, pagination):
+    current_page = 0
+    size = len(query_set)
```

Comment:
```suggestion
    size = query_set.count()
```
This will use SQL's `COUNT` function so you don't have to load the whole queryset into memory (the point of pagination).

Reply:
Good catch. Changed to use this.